components/infusionsoft/sources/common.ts

```diff
@@ -0,0 +1,93 @@
+import infusionsoft from "../app/infusionsoft.app";
+import { SourceHttpRunOptions } from "@pipedream/types";
```

Comment:
That's likely because it was a recent addition to the types package, on `v0.1.4` - previously on `v0.1.3` there was only `SourceRunOptions`. 

https://github.com/PipedreamHQ/pipedream/blob/9a72c1c141865b4126e3e0096452e5c068666ee4/types/src/index.ts#L239

I don't get this error, SourceHttpRunOptions is correctly typed for me - you might need to `npm run build` to build that TS package update